### ISSUE TITLE
text examples

### ISSUE THREAD

**Issue by zcream** [gheaeckkseqrz/SVG2PNG#1]
A major issue with nanosvg is text support. Is there an example of your library supporting text?

**Comment by gheaeckkseqrz** [comment]
No, this is a very basic piece of code, it was used to render very simple graphics, never tried to do anything fancy with it.
If nanosvg doesn't support text, this will not either, all it does is calling ```nsvgRasterize``` and saving the resulting image as a png.
